### ISSUE TITLE
jenkins/inbound-agent:jdk8-nanoserver-1809 - password for this account has expired

### ISSUE THREAD

**Issue by ssallmen** [jenkinsci/docker-inbound-agent#183]
Seems that the same issue as was reported in last May (https://github.com/jenkinsci/docker-inbound-agent/issues/152) for jenkins/jnlp-agent:latest-windows is now affecting at least jenkins/inbound-agent:jdk8-nanoserver-1809 image.

I started to get this error when launching the container:
```
Error: failed to start container "jnlp": Error response from daemon: container jnlp encountered an error during CreateProcess: failure in a Windows system call: The password for this account has expired. (0x532) extra info: {"CommandLine":"pwsh.exe -f C:/ProgramData/Jenkins/jenkins-agent.ps1","User":"jenkins","WorkingDirectory":"C:\\Users\\jenkins","Environment": ...
```

Is there some workaround for this or does it always require some fix in the image?

**Comment by bogdanalov-sw** [comment]
same is happening for me with the "jenkins/inbound-agent:windowsservercore-1809" image

**Comment by rkivisto** [comment]
I noticed a workaround was mentioned by another user, which is to pull down the https://github.com/jenkinsci/docker-inbound-agent repo and build it yourself, as mentioned here: https://github.com/jenkinsci/docker-inbound-agent/issues/173#issuecomment-640762269

Looking at the Pipeline code for this repo, the Windows Docker image is built is by running the following command on a Windows machine that has Docker for Windows installed on it, as per:
https://github.com/jenkinsci/docker-inbound-agent/blob/98527ae42595ca522ab48de014c149fff64ea040/Jenkinsfile#L32

Also see: https://github.com/jenkinsci/docker-inbound-agent/issues/173#issuecomment-655658952

**Comment by ssallmen** [comment]
Yup, got the image built and it seems that it works!
I just had to edit the Dockerfile for jdk8 to use tag jdk8-windowsservercore-1809 instead of tag 4.3-4-windowsservercore-1809 of image jenkins/agent and run the make.ps1. I built only the jdk8 version of the image, which I needed. Need to check if there is a need to open an issue on jenkins/agent itself to get the issue fixed so that the password would not expire, if possible at all.
